README.md

```diff
@@ -98,6 +98,13 @@ details.
 **NOTE:** Generators run in RSpec 3.x will now require `rails_helper` instead
 of `spec_helper`.
 
+### `rails_helper` vs `spec_helper`
+As described [in this stackoverflow answer](http://stackoverflow.com/a/24145760/495637), `spec_helper.rb` is for specs which don't depend on Rails (such as specs for classes in the lib directory). `rails_helper.rb` is for specs which do depend on Rails (in a Rails project, most or all of them).
+
+If you want your non-Rails-dependent specs to enforce that they're non-Rails-dependent, and to run as fast as possible when you run them by themselves, you could require `spec_helper.rb` rather than `rails_helper.rb` in those.
```

Comment:
That is to say, it's redundant to both explicitly `require "spec_helper"` and have `.rspec` require spec_helper?

It's also redundant to have `.rspec` require spec_helper while explicitly requiring `rails_helper` since rails_helper also explicitly requires `spec_helper`.

The specifics of _how_ you require the appropriate files where are fine to leave up to the developer (ie explicitly vs in `.rspec` file)--what I aim to explain here is _why_ you might only require `spec_helper` in some cases. And if there are no cases where a Rails developer might only require `spec_helper`, then I propose that `spec_helper` be merged with `rails_helper` since merging the files would obviate the need to explain why there are two files in the first place.